e2e_tests/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "e2e-tests"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+pytest-playwright-visual = "^2.1.2"
+pytest-playwright = "^0.5.0"
+PyJWT = "^2.8.0"
+itsdangerous = "^2.2.0"
+Flask = "^3.0.3"
```

Comment:
why do we need flask?

Reply:
Can't remember while I added it but don't think its required to run the tests, I've removed it